backend/wmg/pipeline/summary_cubes/calculate_markers.py

```diff
@@ -101,6 +103,17 @@ def _query_tiledb_context_memoized(
     query = q.expression_summary_fmg(criteria)
     cell_counts_query = q.cell_counts(criteria)
 
+    healthy_filter_es = query["disease_ontology_term_id"].astype("str") == NORMAL_CELL_DISEASE_ONTOLOGY_TERM_ID
```

Comment:
The healthy cell filters were moved here. Previously, we were filtering out unhealthy cells from the expression summary fmg cube. That broke parity between the expressions used for calculating marker genes and the cell counts, since cell counts cube was not filtered as it is being shared between WMG and FMG. 

Now, we are explicitly filtering both cell counts and expression summary down to just the healthy cells in the marker gene calculation code.

Reply:
Context would be nice in a method comment in the code